src/nbsap/tests/objectives.py

```diff
@@ -212,29 +213,29 @@ class ObjectivesTest(BaseWebTest):
     def setUp(self):
         NationalActionFactory.reset_sequence()
 
+    @skip
```

Comment:
That page doesn't exist at the moment. I didn't want to remove the test until I was sure that the page will not be used anymore.